tests/run-make/simple-rlib/rmake.rs

```diff
@@ -0,0 +1,13 @@
+// A simple test, which builds and runs foo.rs, which has a dependency
+// on the rlib (a static Rust-specific library format) bar.rs. If the test passes,
+// rlibs are successfully supported by the compiler.
```

Comment:
Nit: this is more like a basic sanity test, it only establishes that a simple rlib can be (1) built and (2) linked into another file.